code/persistentstorage/Databases.md

```diff
@@ -68,7 +67,18 @@ Reference:
 
 #### Kademlia Routing Table
 
-Kademlia Routing Table is a BoltDB key/value store. The routing table is represented by a kademlia key/value store and a node key/value store in one BoltDB instance.
+Kademlia Routing Table is a BoltDB key/value store. The routing table is represented by two bolt buckets ("bucket" in bolt is the same as "table"): 1) a kademlia key/value store (i.e. bolt bucket) and 2) a node key/value store (i.e. bolt bucket) in one BoltDB instance.
+
+The kademlia bolt bucket stores the following data:
+- key: k-bucket IDs (k-buckets are groupings of nodeIDs based on prefix, read original [kademlia paper](https://pdos.csail.mit.edu/~petar/papers/maymounkov-kademlia-lncs.pdf) for details of k-buckets)
+- value: timestamp
+
+The node bolt bucket stores the following data:
+- key: nodeIDs
+- value: contact info for that node
+
+Resources:
+- [Slide deck](https://docs.google.com/presentation/d/1jY9mu8SVmVqirbo_yFIJjHWbeHrEc1NxWkFS1iauT_I/edit#slide=id.p) to a great talk [Jennifer](https://github.com/jenlij) did on storj Kademlia implementation.
```

Comment:
Is that presentation shared with the public?

Reply:
i dont think so

Reply:
We probably dont want to add it here then. Or ask @jenlij to share it with the world 👍

Reply:
@jenlij is that slide deck safe to share with the world? i think it would be a great resource to link here.

Reply:
https://docs.google.com/presentation/d/1jY9mu8SVmVqirbo_yFIJjHWbeHrEc1NxWkFS1iauT_I/edit?usp=sharing

Here is a shareable link!